Assets/UI/Panels/citypanel.lua

```diff
@@ -630,48 +672,59 @@ function ViewMain( data:table )
     else
         productionHash = currentProductionHash;
     end
-    local currentProductionInfo :table = GetProductionInfoOfCity( data.City, productionHash );
 
+    local currentProductionInfo :table = GetProductionInfoOfCity( data.City, productionHash );
```

Comment:
code unchanged in this large section, only the indent was changed.  For whatever reason this part was only indented 2 spaces instead of the expected 4